README.md

```diff
@@ -7,6 +7,16 @@
 
 Continuation Labs linter based on ESLint. See `.eslintrc` file in project root for ESLint settings.
 
+## Badge
```

Comment:
This should definitely go below the important stuff like how to use the tool :-)